assets/pages/new-contributions-landing/components/ContributionThankYouContainer.jsx

```diff
@@ -2,38 +2,40 @@
 
 // ----- Imports ----- //
 
+import type { PaymentMethod } from 'helpers/contributions';
 import React from 'react';
 import { connect } from 'react-redux';
-
 import { type ThankYouPageStageMap, type ThankYouPageStage } from '../contributionsLandingReducer';
 import ContributionThankYou from './ContributionThankYou';
-import SetPassword from './SetPassword/SetPassword';
+import ContributionThankYouSetPassword from './ContributionThankYouSetPassword';
+import ContributionThankYouPasswordSet from './ContributionThankYouPasswordSet';
 
 // ----- Types ----- //
 
 /* eslint-disable react/no-unused-prop-types */
 type PropTypes = {
   thankYouPageStage: ThankYouPageStage,
+  paymentMethod: PaymentMethod,
 };
 /* eslint-enable react/no-unused-prop-types */
 
 const mapStateToProps = state => ({
   thankYouPageStage: state.page.form.thankYouPageStage,
+  paymentMethod: state.page.form.paymentMethod,
 });
 
-// TODO: change landing page copy if password has/hasn't been set
-const thankYouPageStage: ThankYouPageStageMap<React$Element<*>> = {
-  setPassword: (<SetPassword />),
-  thankYou: (<ContributionThankYou />),
-  thankYouPasswordSet: (<ContributionThankYou />),
-  thankYouPasswordNotSet: (<ContributionThankYou />),
-};
-
-
 // ----- Render ----- //
 
 function ContributionThankYouContainer(props: PropTypes) {
 
+  const thankYouPageStage: ThankYouPageStageMap<React$Element<*>> = {
+    thankYou: (<ContributionThankYou showDirectDebitCopy={props.paymentMethod === 'DirectDebit'} />),
```

Comment:
oh, I think we can get rid of `showDirectDebitCopy={props.paymentMethod === 'DirectDebit'}` now, no?

Reply:
... and therefore also remove `paymentMethod` from the props here

Reply:
ah yes good point. 

This has made me think that we should define PropTypes as an [exact object type](https://flow.org/en/docs/types/objects/#toc-exact-object-types) as convention. We already do this in some places, not all. If it had been, flow would have caught this. I will do this in a separate PR.